homeassistant/components/mqtt/mixins.py

```diff
@@ -392,7 +395,7 @@ def attributes_message_received(msg: ReceiveMessage) -> None:
                     }
                     self._attr_extra_state_attributes = filtered_dict
                     get_mqtt_data(self.hass).state_write_requests.write_state_request(
-                        self
+                        self  # type: ignore[arg-type]
```

Comment:
Self is `MqttAttributes` here, but that is a part of `MqttEntity`. As at most other spots `MqttEntity` is reference we can ignore the type here.